src/rebar_fetch.erl

```diff
@@ -61,7 +61,7 @@ download_source(AppDir, Source, State) ->
     catch
         C:T ->
             ?DEBUG("rebar_fetch exception ~p ~p ~p", [C, T, erlang:get_stacktrace()]),
-            throw(?PRV_ERROR({fetch_fail, Source}))
+            ?PRV_ERROR({fetch_fail, Source})
```

Comment:
`download_source/3` specs says it might return `{error, _}` (it can't) and in the two places it is called then error tuple results in a throw. Given that the catch is in the same module as those throws I found the logic more suitable.